### PR TITLE
Fragment animation

### PR DESCRIPTION
**Pull request by ikechiafrics**
Added an update password button in the profile fragment, so the user can change password.
Implemented a swipe navigation feature.

https://user-images.githubusercontent.com/93938295/179131093-423387a8-7109-473a-a8d5-4d5f5dea36d3.mov

The swipe Navigation feature enables the user to move between fragments and tabs by swiping on the screen.
Also, in i added a feature where the user can take a picture and set it as his/her profile picture.

### CODE REVIEW

File: app/src/main/java/com/example/metachef/Adapters/ViewPagerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentPagerAdapter;
+
+import java.util.ArrayList;
+
+public class ViewPagerAdapter extends FragmentPagerAdapter {
```
Comment (lineilfb) [inline app/src/main/java/com/example/metachef/Adapters/ViewPagerAdapter.java]: fyi FragmentPagerAdapter is deprecated https://developer.android.com/reference/androidx/fragment/app/FragmentPagerAdapter
recommended is to use ViewPager2(https://developer.android.com/reference/androidx/viewpager2/widget/ViewPager2)

no need to change now  - maybe for future projects

**Review by lineilfb — APPROVED** [review]

File: app/src/main/java/com/example/metachef/Adapters/ViewPagerAdapter.java (hunk 1 above)
Comment (lineilfb) [reply app/src/main/java/com/example/metachef/Adapters/ViewPagerAdapter.java]: wait is this file not used?

Comment (ikechiafrics) [reply app/src/main/java/com/example/metachef/Adapters/ViewPagerAdapter.java]: it was not being used, I just deleted it

File: app/src/main/java/com/example/metachef/Fragments/ProfileFragment.java [hunk 2]
```diff
@@ -67,9 +69,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-        btnFav = view.findViewById(R.id.btnFav);
-        btnLogout = view.findViewById(R.id.BtnLogout);
+
+        dialog = new Dialog(getContext());
+        dialog.setContentView(R.layout.custom_dialog);
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP){
```
Comment (JamesMcGaa) [inline app/src/main/java/com/example/metachef/Fragments/ProfileFragment.java:75]: what was this for?

Comment (ikechiafrics) [reply app/src/main/java/com/example/metachef/Fragments/ProfileFragment.java:75]: I do not remember, I removed it after i noticed it.

**Pull request merged**